### DISCUSSION TITLE
no dist folder created

### DISCUSSION THREAD

**Discussion by Tommy-T1984** [carla-simulator/carla#4441] [Q&A]
osm2odr.lib(GeoConvHelper.obj) : error LNK2001: 无法解析的外部符号 proj_create
osm2odr.lib(GeoConvHelper.obj) : error LNK2001: 无法解析的外部符号 proj_destroy
osm2odr.lib(GeoConvHelper.obj) : error LNK2001: 无法解析的外部符号 proj_trans
osm2odr.lib(GeoConvHelper.obj) : error LNK2001: 无法解析的外部符号 proj_torad
osm2odr.lib(GeoConvHelper.obj) : error LNK2001: 无法解析的外部符号 proj_todeg
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: static void * __cdecl xercesc_3_2::XMemory::operator new(unsigned __int64)" (??2XMemory@xercesc_3_2@@SAPEAX_K@Z)
osm2odr.lib(XMLSubSys.obj) : error LNK2001: 无法解析的外部符号 "public: static void * __cdecl xercesc_3_2::XMemory::operator new(unsigned __int64)" (??2XMemory@xercesc_3_2@@SAPEAX_K@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static void * __cdecl xercesc_3_2::XMemory::operator new(unsigned __int64)" (??2XMemory@xercesc_3_2@@SAPEAX_K@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static void __cdecl xercesc_3_2::XMemory::operator delete(void *)" (??3XMemory@xercesc_3_2@@SAXPEAX@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: static void __cdecl xercesc_3_2::XMemory::operator delete(void *)" (??3XMemory@xercesc_3_2@@SAXPEAX@Z)
osm2odr.lib(XMLSubSys.obj) : error LNK2001: 无法解析的外部符号 "public: static void __cdecl xercesc_3_2::XMemory::operator delete(void *)" (??3XMemory@xercesc_3_2@@SAXPEAX@Z)
osm2odr.lib(OptionsLoader.obj) : error LNK2001: 无法解析的外部符号 "public: static void __cdecl xercesc_3_2::XMemory::operator delete(void *)" (??3XMemory@xercesc_3_2@@SAXPEAX@Z)
osm2odr.lib(GenericSAXHandler.obj) : error LNK2001: 无法解析的外部符号 "public: static void __cdecl xercesc_3_2::XMemory::operator delete(void *)" (??3XMemory@xercesc_3_2@@SAXPEAX@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: __cdecl xercesc_3_2::SAXParser::SAXParser(class xercesc_3_2::XMLValidator * const,class xercesc_3_2::MemoryManager * const,class xercesc_3_2::XMLGrammarPool * const)" (??0SAXParser@xercesc_3_2@@QEAA@QEAVXMLValidator@1@QEAVMemoryManager@1@QEAVXMLGrammarPool@1@@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual __cdecl xercesc_3_2::SAXParser::~SAXParser(void)" (??1SAXParser@xercesc_3_2@@UEAA@XZ)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: void __cdecl xercesc_3_2::SAXParser::setDoNamespaces(bool)" (?setDoNamespaces@SAXParser@xercesc_3_2@@QEAAX_N@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: void __cdecl xercesc_3_2::SAXParser::setValidationScheme(enum xercesc_3_2::SAXParser::ValSchemes)" (?setValidationScheme@SAXParser@xercesc_3_2@@QEAAXW4ValSchemes@12@@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: void __cdecl xercesc_3_2::SAXParser::setDoSchema(bool)" (?setDoSchema@SAXParser@xercesc_3_2@@QEAAX_N@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: bool __cdecl xercesc_3_2::SAXParser::parseFirst(class xercesc_3_2::InputSource const &,class xercesc_3_2::XMLPScanToken &)" (?parseFirst@SAXParser@xercesc_3_2@@QEAA_NAEBVInputSource@2@AEAVXMLPScanToken@2@@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: bool __cdecl xercesc_3_2::SAXParser::parseNext(class xercesc_3_2::XMLPScanToken &)" (?parseNext@SAXParser@xercesc_3_2@@QEAA_NAEAVXMLPScanToken@2@@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual void __cdecl xercesc_3_2::SAXParser::parse(char const * const)" (?parse@SAXParser@xercesc_3_2@@UEAAXQEBD@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual void __cdecl xercesc_3_2::SAXParser::setDocumentHandler(class xercesc_3_2::DocumentHandler * const)" (?setDocumentHandler@SAXParser@xercesc_3_2@@UEAAXQEAVDocumentHandler@2@@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual void __cdecl xercesc_3_2::SAXParser::setErrorHandler(class xercesc_3_2::ErrorHandler * const)" (?setErrorHandler@SAXParser@xercesc_3_2@@UEAAXQEAVErrorHandler@2@@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual __cdecl xercesc_3_2::BinInputStream::~BinInputStream(void)" (??1BinInputStream@xercesc_3_2@@UEAA@XZ)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: virtual __cdecl xercesc_3_2::BinInputStream::~BinInputStream(void)" (??1BinInputStream@xercesc_3_2@@UEAA@XZ)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual char16_t const * __cdecl xercesc_3_2::BinInputStream::getEncoding(void)const " (?getEncoding@BinInputStream@xercesc_3_2@@UEBAPEB_SXZ)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: virtual char16_t const * __cdecl xercesc_3_2::BinInputStream::getEncoding(void)const " (?getEncoding@BinInputStream@xercesc_3_2@@UEBAPEB_SXZ)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "protected: __cdecl xercesc_3_2::BinInputStream::BinInputStream(void)" (??0BinInputStream@xercesc_3_2@@IEAA@XZ)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "protected: __cdecl xercesc_3_2::BinInputStream::BinInputStream(void)" (??0BinInputStream@xercesc_3_2@@IEAA@XZ)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual __cdecl xercesc_3_2::InputSource::~InputSource(void)" (??1InputSource@xercesc_3_2@@UEAA@XZ)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: virtual __cdecl xercesc_3_2::InputSource::~InputSource(void)" (??1InputSource@xercesc_3_2@@UEAA@XZ)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual void __cdecl xercesc_3_2::InputSource::setEncoding(char16_t const * const)" (?setEncoding@InputSource@xercesc_3_2@@UEAAXQEB_S@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: virtual void __cdecl xercesc_3_2::InputSource::setEncoding(char16_t const * const)" (?setEncoding@InputSource@xercesc_3_2@@UEAAXQEB_S@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual void __cdecl xercesc_3_2::InputSource::setPublicId(char16_t const * const)" (?setPublicId@InputSource@xercesc_3_2@@UEAAXQEB_S@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: virtual void __cdecl xercesc_3_2::InputSource::setPublicId(char16_t const * const)" (?setPublicId@InputSource@xercesc_3_2@@UEAAXQEB_S@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: virtual void __cdecl xercesc_3_2::InputSource::setSystemId(char16_t const * const)" (?setSystemId@InputSource@xercesc_3_2@@UEAAXQEB_S@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: virtual void __cdecl xercesc_3_2::InputSource::setSystemId(char16_t const * const)" (?setSystemId@InputSource@xercesc_3_2@@UEAAXQEB_S@Z)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "protected: __cdecl xercesc_3_2::InputSource::InputSource(char const * const,class xercesc_3_2::MemoryManager * const)" (??0InputSource@xercesc_3_2@@IEAA@QEBDQEAVMemoryManager@1@@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "protected: __cdecl xercesc_3_2::InputSource::InputSource(char const * const,class xercesc_3_2::MemoryManager * const)" (??0InputSource@xercesc_3_2@@IEAA@QEBDQEAVMemoryManager@1@@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static class xercesc_3_2::MemoryManager * xercesc_3_2::XMLPlatformUtils::fgMemoryManager" (?fgMemoryManager@XMLPlatformUtils@xercesc_3_2@@2PEAVMemoryManager@2@EA)
osm2odr.lib(SUMOSAXAttributesImpl_Xerces.obj) : error LNK2001: 无法解析的外部符号 "public: static class xercesc_3_2::MemoryManager * xercesc_3_2::XMLPlatformUtils::fgMemoryManager" (?fgMemoryManager@XMLPlatformUtils@xercesc_3_2@@2PEAVMemoryManager@2@EA)
osm2odr.lib(OptionsIO.obj) : error LNK2001: 无法解析的外部符号 "public: static class xercesc_3_2::MemoryManager * xercesc_3_2::XMLPlatformUtils::fgMemoryManager" (?fgMemoryManager@XMLPlatformUtils@xercesc_3_2@@2PEAVMemoryManager@2@EA)
osm2odr.lib(XMLSubSys.obj) : error LNK2001: 无法解析的外部符号 "public: static class xercesc_3_2::MemoryManager * xercesc_3_2::XMLPlatformUtils::fgMemoryManager" (?fgMemoryManager@XMLPlatformUtils@xercesc_3_2@@2PEAVMemoryManager@2@EA)
osm2odr.lib(StringUtils.obj) : error LNK2001: 无法解析的外部符号 "public: static class xercesc_3_2::MemoryManager * xercesc_3_2::XMLPlatformUtils::fgMemoryManager" (?fgMemoryManager@XMLPlatformUtils@xercesc_3_2@@2PEAVMemoryManager@2@EA)
osm2odr.lib(GenericSAXHandler.obj) : error LNK2001: 无法解析的外部符号 "public: static class xercesc_3_2::MemoryManager * xercesc_3_2::XMLPlatformUtils::fgMemoryManager" (?fgMemoryManager@XMLPlatformUtils@xercesc_3_2@@2PEAVMemoryManager@2@EA)
osm2odr.lib(XMLSubSys.obj) : error LNK2001: 无法解析的外部符号 "public: static void __cdecl xercesc_3_2::XMLPlatformUtils::Initialize(char const * const,char const * const,class xercesc_3_2::PanicHandler * const,class xercesc_3_2::MemoryManager * const)" (?Initialize@XMLPlatformUtils@xercesc_3_2@@SAXQEBD0QEAVPanicHandler@2@QEAVMemoryManager@2@@Z)
osm2odr.lib(XMLSubSys.obj) : error LNK2001: 无法解析的外部符号 "public: static void __cdecl xercesc_3_2::XMLPlatformUtils::Terminate(void)" (?Terminate@XMLPlatformUtils@xercesc_3_2@@SAXXZ)
osm2odr.lib(XMLSubSys.obj) : error LNK2001: 无法解析的外部符号 "public: static class xercesc_3_2::SAX2XMLReader * __cdecl xercesc_3_2::XMLReaderFactory::createXMLReader(class xercesc_3_2::MemoryManager * const,class xercesc_3_2::XMLGrammarPool * const)" (?createXMLReader@XMLReaderFactory@xercesc_3_2@@SAPEAVSAX2XMLReader@2@QEAVMemoryManager@2@QEAVXMLGrammarPool@2@@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static class xercesc_3_2::SAX2XMLReader * __cdecl xercesc_3_2::XMLReaderFactory::createXMLReader(class xercesc_3_2::MemoryManager * const,class xercesc_3_2::XMLGrammarPool * const)" (?createXMLReader@XMLReaderFactory@xercesc_3_2@@SAPEAVSAX2XMLReader@2@QEAVMemoryManager@2@QEAVXMLGrammarPool@2@@Z)
osm2odr.lib(XMLSubSys.obj) : error LNK2001: 无法解析的外部符号 "public: __cdecl xercesc_3_2::XMLGrammarPoolImpl::XMLGrammarPoolImpl(class xercesc_3_2::MemoryManager * const)" (??0XMLGrammarPoolImpl@xercesc_3_2@@QEAA@QEAVMemoryManager@1@@Z)
osm2odr.lib(XMLSubSys.obj) : error LNK2001: 无法解析的外部符号 "public: static char16_t const * const xercesc_3_2::XMLUni::fgXercesHandleMultipleImports" (?fgXercesHandleMultipleImports@XMLUni@xercesc_3_2@@2QB_SB)
osm2odr.lib(XMLSubSys.obj) : error LNK2001: 无法解析的外部符号 "public: static char const * const xercesc_3_2::XMLUni::fgXercescDefaultLocale" (?fgXercescDefaultLocale@XMLUni@xercesc_3_2@@2QBDB)
osm2odr.lib(OptionsLoader.obj) : error LNK2001: 无法解析的外部符号 "public: __cdecl xercesc_3_2::SAXParseException::SAXParseException(class xercesc_3_2::SAXParseException const &)" (??0SAXParseException@xercesc_3_2@@QEAA@AEBV01@@Z)
osm2odr.lib(GenericSAXHandler.obj) : error LNK2001: 无法解析的外部符号 "public: __cdecl xercesc_3_2::SAXParseException::SAXParseException(class xercesc_3_2::SAXParseException const &)" (??0SAXParseException@xercesc_3_2@@QEAA@AEBV01@@Z)
osm2odr.lib(OptionsLoader.obj) : error LNK2001: 无法解析的外部符号 "public: virtual __cdecl xercesc_3_2::SAXParseException::~SAXParseException(void)" (??1SAXParseException@xercesc_3_2@@UEAA@XZ)
osm2odr.lib(GenericSAXHandler.obj) : error LNK2001: 无法解析的外部符号 "public: virtual __cdecl xercesc_3_2::SAXParseException::~SAXParseException(void)" (??1SAXParseException@xercesc_3_2@@UEAA@XZ)
osm2odr.lib(OptionsLoader.obj) : error LNK2001: 无法解析的外部符号 "public: unsigned __int64 __cdecl xercesc_3_2::SAXParseException::getColumnNumber(void)const " (?getColumnNumber@SAXParseException@xercesc_3_2@@QEBA_KXZ)
osm2odr.lib(GenericSAXHandler.obj) : error LNK2001: 无法解析的外部符号 "public: unsigned __int64 __cdecl xercesc_3_2::SAXParseException::getColumnNumber(void)const " (?getColumnNumber@SAXParseException@xercesc_3_2@@QEBA_KXZ)
osm2odr.lib(OptionsLoader.obj) : error LNK2001: 无法解析的外部符号 "public: unsigned __int64 __cdecl xercesc_3_2::SAXParseException::getLineNumber(void)const " (?getLineNumber@SAXParseException@xercesc_3_2@@QEBA_KXZ)
osm2odr.lib(GenericSAXHandler.obj) : error LNK2001: 无法解析的外部符号 "public: unsigned __int64 __cdecl xercesc_3_2::SAXParseException::getLineNumber(void)const " (?getLineNumber@SAXParseException@xercesc_3_2@@QEBA_KXZ)
osm2odr.lib(StringUtils.obj) : error LNK2001: 无法解析的外部符号 "public: unsigned char * __cdecl xercesc_3_2::ArrayJanitor<unsigned char>::get(void)const " (?get@?$ArrayJanitor@E@xercesc_3_2@@QEBAPEAEXZ)
osm2odr.lib(StringUtils.obj) : error LNK2001: 无法解析的外部符号 "public: __cdecl xercesc_3_2::TranscodeToStr::TranscodeToStr(char16_t const *,char const *,class xercesc_3_2::MemoryManager *)" (??0TranscodeToStr@xercesc_3_2@@QEAA@PEB_SPEBDPEAVMemoryManager@1@@Z)
osm2odr.lib(StringUtils.obj) : error LNK2001: 无法解析的外部符号 "public: __cdecl xercesc_3_2::TranscodeToStr::~TranscodeToStr(void)" (??1TranscodeToStr@xercesc_3_2@@QEAA@XZ)
osm2odr.lib(GenericSAXHandler.obj) : error LNK2001: 无法解析的外部符号 "public: static char * __cdecl xercesc_3_2::XMLString::transcode(char16_t const * const,class xercesc_3_2::MemoryManager * const)" (?transcode@XMLString@xercesc_3_2@@SAPEADQEB_SQEAVMemoryManager@2@@Z)
osm2odr.lib(GenericSAXHandler.obj) : error LNK2001: 无法解析的外部符号 "public: static void __cdecl xercesc_3_2::XMLString::release(char * *,class xercesc_3_2::MemoryManager * const)" (?release@XMLString@xercesc_3_2@@SAXPEAPEADQEAVMemoryManager@2@@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: __cdecl xercesc_3_2::MemBufInputSource::MemBufInputSource(unsigned char const * const,unsigned __int64,char const * const,bool,class xercesc_3_2::MemoryManager * const)" (??0MemBufInputSource@xercesc_3_2@@QEAA@QEBE_KQEBD_NQEAVMemoryManager@1@@Z)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: virtual __cdecl xercesc_3_2::MemBufInputSource::~MemBufInputSource(void)" (??1MemBufInputSource@xercesc_3_2@@UEAA@XZ)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static char16_t const * const xercesc_3_2::XMLUni::fgWFXMLScanner" (?fgWFXMLScanner@XMLUni@xercesc_3_2@@2QB_SB)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static char16_t const * const xercesc_3_2::XMLUni::fgIGXMLScanner" (?fgIGXMLScanner@XMLUni@xercesc_3_2@@2QB_SB)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static char16_t const * const xercesc_3_2::XMLUni::fgXercesDynamic" (?fgXercesDynamic@XMLUni@xercesc_3_2@@2QB_SB)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static char16_t const * const xercesc_3_2::XMLUni::fgXercesSchema" (?fgXercesSchema@XMLUni@xercesc_3_2@@2QB_SB)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static char16_t const * const xercesc_3_2::XMLUni::fgXercesUseCachedGrammarInParse" (?fgXercesUseCachedGrammarInParse@XMLUni@xercesc_3_2@@2QB_SB)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static char16_t const * const xercesc_3_2::XMLUni::fgXercesScannerName" (?fgXercesScannerName@XMLUni@xercesc_3_2@@2QB_SB)
osm2odr.lib(SUMOSAXReader.obj) : error LNK2001: 无法解析的外部符号 "public: static char16_t const * const xercesc_3_2::XMLUni::fgSAX2CoreValidation" (?fgSAX2CoreValidation@XMLUni@xercesc_3_2@@2QB_SB)
osm2odr.lib(SUMOSAXAttributesImpl_Xerces.obj) : error LNK2001: 无法解析的外部符号 "public: static char16_t * __cdecl xercesc_3_2::XMLString::transcode(char const * const,class xercesc_3_2::MemoryManager * const)" (?transcode@XMLString@xercesc_3_2@@SAPEA_SQEBDQEAVMemoryManager@2@@Z)
osm2odr.lib(SUMOSAXAttributesImpl_Xerces.obj) : error LNK2001: 无法解析的外部符号 "public: static void __cdecl xercesc_3_2::XMLString::release(char16_t * *,class xercesc_3_2::MemoryManager * const)" (?release@XMLString@xercesc_3_2@@SAXPEAPEA_SQEAVMemoryManager@2@@Z)
build\lib.win-amd64-3.9\carla\libcarla.cp39-win_amd64.pyd : fatal error LNK1120: 52 个无法解析的外部命令

**Comment by LucasFKobernic** [comment]
Hi @Pt0lemaeus,

I'm having the same issue with Carla 0.9.12. 

Have you solved it?

**Comment by roque2205** [comment]
Hi, it may help to copy the osm2odr-visualstudio folder from the build folder of Carla 0.9.11 into the one in Carla 0.9.12 and rerun "make PythonAPI". Helped for me once, now after making changes to the PythonAPI it won't anymore, so I guess there's about a 50/50 chance. ;)

**Comment by nnorbert99** [comment]
I am having the same issue couldn't solve it yet, i tried to reinstall Visual studio, cmake, make, i tried to git checkout 0.9.11, but the same issue occur.

**Comment by roque2205** [comment]
In case you receive exactly the same errors as in the original post, simply rerunning make PythonAPI should do the trick.
So: make PythonAPI --> failed due to OSM2ODR errors --> make PythonAPI --> enjoy. :)

**Reply by nnorbert99** [reply]
actually not the same error its the same regarding the link 1120 but with the libcarla.obj any idea for that ?

libcarla.obj : error LNK2001: unresolved external symbol "class boost::python::api::object __cdecl boost::python::api::getitem(class boost::python::api::object const &,class boost::python::api::object const &)" (?getitem@api@python@boost@@YA?AVobject@123@AEBV4123@0@Z)
build\lib.win-amd64-3.7\carla\libcarla.cp37-win_amd64.pyd : fatal error LNK1120: 77 unresolved externals
error: command 'C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Community\\VC\\Tools\\MSVC\\14.16.27023\\bin\\HostX86\\x64\\link.exe' failed with exit status 1120

**Reply by roque2205** [reply]
I didn't get unresolved externals in the boost class yet. Have you made custom changes to the code?

By the way, apparently you use VS2017 when 0.9.12 requires VS2019. That might be an issue.

**Reply by nnorbert99** [reply]
But i did git checkout 0.9.11 and i am using 0.9.11 assests, so it should work i didnt change any code, just following the provided manual. ( i tried it with the 0.9.12 with vs2019 and the same error occured. Maybe there is some additional dependency i might not knowing ?

**Reply by roque2205** [reply]
Ah, you're on 0.9.11, okay. I never had any issues with that version, 0.9.12 is still a bit buggy though.

If you followed every single step of the build instructions on readthedocs I suppose 0.9.11 should work nicely. :/

**Reply by nnorbert99** [reply]
yea and it worked for me once, this is a reinstall but its not working now. i dont know what to do.
